dev/check-style.sh

```diff
@@ -1,22 +0,0 @@
-#!/bin/bash
-
-set -exuo pipefail
-
-cd "$(dirname "$0")"
-cd ../
-
-black --check --diff .
-
-# E203: whitespace before ':' (not compliant with PEP 8)
-# E402: module level import not at top of file
-python3 -m flake8 --exclude=external --extend-ignore=E203,E402 --max-line-length=110 --builtins="_" rednotebook tests setup.py dev/whitelist.py
-
-isort --check-only rednotebook/ tests/
-
-python3 -m pyupgrade --py36-plus `find rednotebook tests -name "*.py" -not -path "*external*"`
```

Comment:
I didn't add this command to the _style_ env in the tox file since `pyupgrade` doesn't have a check only option. That's why I think a command that modifies the code would be better to only be in the _fix-style_ env.

Reply:
I think it's important to have pyupgrade in the "style" env. I only call fix-style if style fails, for example.

Reply:
Ok. Ill add it back.

My only problem is the command looks ugly and is a little bit unclear at first glance what it is doing. Plus I don't think is compatible with windows. Here, https://github.com/asottile/pyupgrade/issues/40, the creator of pyupgrade suggest using _pre-commit_ to run `pyupgrade`. Do you think it is convenient to handle this with _pre-commit_? I am not familiar with the tool yet so I really can't tell. If you consider it a good idea, I can make the arrangements for this to work with _pre-commit_, on this PR.

Reply:
I'm using pre-commit locally and it's quite useful. Not sure how difficult it is to set it up within tox, though. Feel free to try, would be nice to get rid of the ugly command version.